static/main.js

```diff
@@ -0,0 +1,28 @@
+const notificationForm = document.getElementById('notificationForm')
+notificationForm.addEventListener('submit', function(event){
+    event.preventDefault();
+    createNotification();
+});
+
+function createNotification(){
+    const form = notificationForm
+    let title = document.getElementById('title').value
+    let description = document.getElementById('description').value
+
+    fetch('http://localhost:8000/notifications', {
+        method: 'POST',
+        body: JSON.stringify({
+            title:title,
+            description:description,
+        }),
+        headers: {
+            'Content-type': 'application/json; charset=UTF-8',
+        }
+    })
+    .then(function(response){
+        return response.json()})
+    .then(function(data){
```

Comment:
Looks good. you can directly use `(variable)=>{} ` also known as arrow function here like you used in the catch block just to maintain the uniformity.

Reply:
Got it. Thanks!